packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -162,7 +184,12 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
         icon:
           'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2Fsimplex.jpg?alt=media&token=6037b2f9-9d76-4076-b29e-b7e0de0b3f34',
         image: <Image source={simplexLogo} style={styles.logo} resizeMode={'contain'} />,
-        onSelected: () => openSimplex(account),
+        isFeeDataLoading: !simplexQuote?.quote_id,
+        onSelected: () => {
+          if (simplexQuote && userIpAddress) {
```

Comment:
If these conditions aren't met, should we show an error message or something? Otherwise it will look like tapping is just not working.

Reply:
We now disable options that we were able to get quotes for